drftest/templates/doc_of_app.md

```diff
@@ -53,9 +53,15 @@ This class has the following test cases:
 
 {% if method_doc.response.data %}
 * **Response data:** 
+{% if method_doc.response.content_type == 'application/json' %}
 ```json
 {{ method_doc.response.data|to_json }}
 ```
+{% else %}
+{% for r in method_doc.response.data %}|{% for c in r %}{{ c }}|{% endfor %}
+|{% for c in r %}|{% endfor %}
```

Comment:
Would you please change c and r to sth more descriptive? I have no idea what they are, well I guess they are cell and row, I think naming them that way would be nicer.

BTW it would be nice if instead of just else we checked for content type to be excel

Reply:
Done 7029dd1